### ISSUE TITLE
IE10 crash and then bad UIA state of address bar after typing certain addresses in address bar with multiple tabs

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1809]
**Reported by kevinchao89 on 2011-09-18 07:24**
Windows 8 x86, NVDA Snapshot Main-4693

Start screen, IE10 Metro app;
Notice that NVDA does not speak title of app that gains focus, Internet Explorer

ALT+D for address bar, requires pressing TAB/moving around UI a bit before NVDA can get focus in address bar, type NVDA-Project.org, and ENTER;
NVDA should be able to get focus into address  bar right away, typing in URL should not read suggestion, making process very verbose, and IE10 Metro app will crash

Bring up IE10 metro app again, ALT+D for address bar, Google.com;
With each press of a character in address bar, NVDA will make error sound.

**Comment by bhavyashah** [comment]
@michaelDCurran As indicated in https://github.com/nvaccess/nvda/issues/1809#issuecomment-155293174, you were able to successfully reproduce the reported crash. Is this still the case, or has this issue been resolved with an updated software setup? If no one is able to replicate this anymore, I suggest closing.

**Comment by Adriani90** [comment]
@michaelDCurran is this still reproducible?

**Comment by josephsl** [comment]
Hi, on Windows 7 and 8.0, that is? Thanks.

**Comment by bdorer** [comment]
I wote for closing as Windows 7 support ends on 2020-01-24 and IE is no longer developed further.

**Comment by leonardder** [comment]
I agree with closing this. If there's any news or someone who can reproduce this, we can always reopen.
